File: tools/SharedJavaLibrary/src/network/message/torobot/StopRemoteControlMessage.java
package network.message.torobot;

import network.message.RemoteMessageToRobot;

public class StopRemoteControlMessage extends RemoteMessageToRobot {

  public StopRemoteControlMessage() {
    super(RemoteMessageToRobot.MessageType.STOP_REMOTE_CONTROL, null);
  }
  
}

File: robot/Comm/Messages/Remote/ToRobot/StopRemoteControlMessage.cpp
#include "StopRemoteControlMessage.h"

#include <cstring>

StopRemoteControlMessage::StopRemoteControlMessage()
  : RemoteMessageToRobot(RemoteMessageToRobot::stopRemoteControl, 0, NULL) {
}

StopRemoteControlMessage::~StopRemoteControlMessage() {
}
